exercises/nth-prime/src/example.c

```diff
@@ -0,0 +1,32 @@
+#include "nth_prime.h"
+#include <stdbool.h>
+#include <math.h>
+
+static bool is_prime(uint32_t n)
+{
+  for(uint32_t i = (uint32_t)sqrt(n); i > 1; i--) {
```

Comment:
If you wanted to get _really_ crazy, you could use C99 variable length arrays and do a real sieve. That should be a fair bit faster, but this is also a nice, simple solution.